warehouse/integrations/bigquery/testhelper/setup.go

```diff
@@ -34,3 +44,35 @@ func IsBQTestCredentialsAvailable() bool {
 	_, err := GetBQTestCredentials()
 	return err == nil
 }
+
+func RecordsFromWarehouse(
```

Comment:
What is this function actually doing? It's hard to tell without reading its code due to its generic name and lack of comment/docs. We should be able to make it more descriptive.